client/blocks/plan-purchase-features/index.jsx

```diff
@@ -0,0 +1,243 @@
+					' its professional features.' ) }
+			/>,
+			<CustomDomainFeature
+				selectedSite={ selectedSite }
+				hasDomainCredit={ plan && plan.hasDomainCredit }
+				key="customDomainFeature"
+			/>,
+			<FindNewThemeFeature
+				selectedSite={ selectedSite }
+				key="findNewThemeFeature"
+			/>,
+			<GoogleAnalyticsStatsFeature
+				selectedSite={ selectedSite }
+				key="googleAnalyticsStatsFeature"
+			/>,
+			<AdvertisingRemovedFeature
+				isBusinessPlan
+				key="advertisingRemovedFeature"
+			/>,
+			<GoogleVouchersFeature
+				selectedSite={ selectedSite }
+				key="googleVouchersFeature"
+			/>,
+			<CustomizeThemeFeature
+				customizeLink={ this.getCustomizeLink() }
+				isCustomizeEnabled={ this.isCustomizeEnabled() }
+				key="customizeThemeFeature"
+			/>,
+			<VideoAudioPostsFeature
+				paths={ paths }
+				selectedSite={ selectedSite }
+				key="videoAudioPostsFeature"
+			/>,
+			isWordadsInstantActivationEligible( selectedSite )
+				? <MonetizeSiteFeature
+					selectedSite={ selectedSite }
+					key="monetizeSiteFeature"
+				/>
+				: null
+		];
+	}
+
+	getPremiumFeatures() {
+		const {
+			selectedSite,
+			sitePlans
+		} = this.props;
+
+		const plan = find( sitePlans.data, isPremium );
+
+		return [
+			<CustomDomainFeature
+				selectedSite={ selectedSite }
+				hasDomainCredit={ plan && plan.hasDomainCredit }
+				key="customDomainFeature"
+			/>,
+			<AdvertisingRemovedFeature
+				isBusinessPlan={ false }
+				key="advertisingRemovedFeature"
+			/>,
+			<GoogleVouchersFeature
+				selectedSite={ selectedSite }
+				key="googleVouchersFeature"
+			/>,
+			<CustomizeThemeFeature
+				customizeLink={ this.getCustomizeLink() }
+				isCustomizeEnabled={ this.isCustomizeEnabled() }
+				key="customizeThemeFeature"
+			/>,
+			<VideoAudioPostsFeature
+				paths={ paths }
+				selectedSite={ selectedSite }
+				key="videoAudioPostsFeature"
+			/>,
+			isWordadsInstantActivationEligible( selectedSite )
+				? <MonetizeSiteFeature
+					selectedSite={ selectedSite }
+					key="monetizeSiteFeature"
+				/>
+				: null
+		];
+	}
+
+	getPersonalFeatures() {
+		const {
+			selectedSite,
+			sitePlans
+		} = this.props;
+
+		const plan = find( sitePlans.data, isPremium );
+
+		return [
+			<CustomDomainFeature
+				selectedSite={ selectedSite }
+				hasDomainCredit={ plan && plan.hasDomainCredit }
+				key="customDomainFeature"
+			/>,
+			<AdvertisingRemovedFeature
+				isBusinessPlan={ false }
+				key="advertisingRemovedFeature"
+			/>
+		];
+	}
+
+	getJetpackFeatures() {
+		const {	selectedSite } = this.props;
+
+		return [
+			<JetpackFeatures
+				selectedSite={ selectedSite }
+				user={ userFactory() }
+				analytics={ analytics }
+				utils={ utils }
+				isPluginsSetupEnabled={ isEnabled( 'manage/plugins/setup' ) }
+				key="jetpackFeatures"
+			/>
+		];
+	}
+
+	getPlanPurchaseFeatures() {
+		const { plan } = this.props;
+
+		switch ( plan ) {
```

Comment:
Sidenote: I love the code reviews, especially for spotting things like this!

Where can I find the legacy plans? How many legacy plans do we support? I tried the API console and `/v1.[1-4]/plans` endpoint which gave me only the listed ones here:

![selection_023](https://cloud.githubusercontent.com/assets/4988512/17674040/15e63262-6324-11e6-8402-e84867086b74.jpg)